core/new-gui/src/app/dashboard/service/workflow-version/workflow-version.service.ts

```diff
@@ -52,9 +60,80 @@ export class WorkflowVersionService {
     this.setDisplayParticularVersion(true);
     // disable modifications because it is read only
     this.workflowActionService.disableWorkflowModification();
+    // highlight the different elements
+    this.highlightDifference(this.highlightedIDs);
+  }
+
+  public getPositionDifference(v1: Workflow, v2: Workflow) {
+    var p1 = v1.content.operatorPositions;
+    var p2 = v2.content.operatorPositions;
+    var difference = reduce(
+      p2,
+      function (result: string[], value, key) {
+        return p1[key as keyof typeof p1] == undefined || isEqual(value, p1[key as keyof typeof p1])
+          ? result
+          : result.concat(key);
+      },
+      []
+    );
+    return difference;
+  }
+
+  public highlightDifference(differences: any[]) {
+    for (var diff of differences) {
+      this.workflowActionService
+        .getJointGraphWrapper()
+        .getMainJointPaper()
+        ?.getModelById(diff)
+        .attr("rect.boundary/fill", "rgba(255,118,20,0.5)");
+    }
+  }
+
+  public getDifference(v1: Workflow, v2: Workflow) {
+    var typeToInclude = ["operators", "links", "groups", "commentBoxes", "breakpoints"];
```

Comment:
let this a configurable list of constants so that we can add more in the future rather than hard code it